semgrep/tests/e2e/snapshots/test_rule_parser/test_rule_parser__failure__error_messages/badlanguage/error.json

```diff
@@ -3,8 +3,8 @@
     {
       "code": 8,
       "level": "error",
-      "long_msg": "unsupported language rust",
-      "short_msg": "invalid language",
+      "long_msg": "unsupported language: rust. supported languages are: C#, JSON, Json, c, cs, csharp, generic, go, golang, java, javascript, js, json, jsx, lua, ml, none, ocaml, php, py, python, python2, python3, rb, regex, ruby, ts, tsx, typescript",
```

Comment:
Couple questions:
1. Is it possible to alphabetize this list to make languages easily searchable/discoverable?
2. Did you consider proper sentence capitalization? 
2. Could we lowercase and dedup items of the list? i.e. JSON vs Json, C# -> c#.
3. How did you choose the rather cold error message tone? ;) Would something more human be acceptable?

> The language `rust` is unsupported. Please choose one of: c, c#, json ...